src/CommitmentsV2/SFA.DAS.CommitmentsV2.Messages/Events/ApprenticeshipResumedEvent.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace SFA.DAS.CommitmentsV2.Messages.Events
+{
+    public class ApprenticeshipResumedEvent
```

Comment:
Naming events so that they have parity with the commands can be a nice convention i.e. `ResumedApprenticeshipCommand` could be published as a result of handling `ResumeApprenticeshipCommand`. What do you think?

Reply:
See previous response